lib/models/ObjectLockConfiguration.js

```diff
@@ -145,39 +145,39 @@ class ObjectLockConfiguration {
                 'request xml does not include valid ObjectLockEnabled');
             return validConfig;
         }
-        const ruleArray = objectLockConfig.Rule;
-        if (ruleArray.length > 1) {
-            validConfig.error = errors.MalformedXML.customizeDescription(
-                'request xml contains more than one rule');
-            return validConfig;
-        }
-
-        const drArray = ruleArray[0].DefaultRetention;
-        if (!drArray || !drArray[0] || drArray[0] === '') {
-            validConfig.error = errors.MalformedXML.customizeDescription(
-                'Rule request xml does not contain DefaultRetention');
-            return validConfig;
-        }
-        if (!drArray[0].Mode || (!drArray[0].Days && !drArray[0].Years)) {
-            validConfig.error = errors.MalformedXML.customizeDescription(
-                'DefaultRetention request xml does not contain Mode or ' +
-                'retention period (Days or Years)');
-            return validConfig;
-        }
-        const validMode = this._parseMode(drArray[0].Mode);
-        if (validMode.error) {
-            validConfig.error = validMode.error;
-            return validConfig;
+        if (objectLockConfig.Rule) {
+            const ruleArray = objectLockConfig.Rule;
+            if (ruleArray && ruleArray.length > 1) {
```

Comment:
`ruleArray &&` is not needed since it is checked by `if (objectLockConfig.Rule)`